modules/app/recording/dashboard.tsx

```diff
@@ -103,7 +136,9 @@ const Dashboard = () => {
 
   const handleUpload = () => {
     if (selectedFile) {
+      setIsUploading(true);
```

Comment:
I think this upload starts from before, here is already uploaded? (thats why we can not see the spinner)